source/reference/limitations.txt

```diff
@@ -150,6 +150,9 @@ Sharded Clusters
   returns an error and stops, which requires you to start the migration
   from scratch.
 
+- ``mongosync`` doesn't support running the command :dbcommand:`transitionFromDedicatedConfigServer`
+  during ``mongosync`` execution.
```

Comment:
[nit] to avoid redundancy I think we can take out the second mention of `mongosync` here!